python/lsst/ts/tunablelaser/hardware.py

```diff
@@ -201,7 +203,7 @@ def start_propagating(self):
         self.power_register_2.set_register_value("ON")
 
     def stop_propagating(self):
-        """Stops the propagation of the laser
+        """Stop the propagation of the laser
```

Comment:
Please document what happens if the laser is not propagating when you call this.